test/examples.js

```diff
@@ -4,7 +4,7 @@ const { tspl } = require('@matteo.collina/tspl')
 const { createServer } = require('node:http')
 const { test, after } = require('node:test')
 const { once } = require('node:events')
-const examples = require('../examples/request.js')
+const examples = require('../documentation/examples/request.js')
```

Comment:
What is the point of this test? Only to ensure that th documentation is right?

Maybe it is wrong to extract examples and move them to documentation?

Reply:
Maybe the github workflow for unit tests should trigger on changes when files in examples folder are changed?

So when documentation is touched no codecoverage has to run? 

Maybe move the examples test also into documentation folder?

Reply:
I think all good ideas. Not sure if this should block, but definitely should be improved. WDYT? Can we land as is now and work on this later?

Reply:
yes, it can be optimized later